feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -0,0 +1,127 @@
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.office.R
+
+/**
+ * This class will handle the network states responses, i modification with the structure.
+ * @param onClickItem click back when you do it on an item
+ * @param onSuccess our event that send us to the next view
+ * TODO implement @param viewModel the view model we are using inside our view compose has to be injected
+ * /*viewModel: OfficeViewModel = hiltViewModel()*/
+ * @author Andres Ivan Medina
+ */
+@Composable
+fun OfficeView() {
+    val viewModel = OfficeViewModel(
+        IntentActionsImpl(
+            LocalContext.current, PermissionsProviderImpl(
+                LocalContext.current
+            )
+        )
+    )
+    LaunchedEffect(true) {
+        viewModel.loadOfficeInformationData()
+    }
+    val officeDataList = remember {
+        viewModel.dataOfficeInfo
+    }
+    OfficeUI(
+        officeDataList,
+        onCallButtonClicked = { viewModel.makeCall() },
+        onEmailButtonClicked = { viewModel.sendEmail() }
+    )
+}
+
+/**
+ * This function creates the Office screen UI
+ * @param onCallButtonClicked launches the call intent
+ * @param onEmailButtonClicked launches the Email intent
+ * @author Yaritza Moreno
+ * Modified by:
+ * @author Andres Ivan Medina
+ */
+@Composable
+fun OfficeUI(
+    list: List<OfficeInfoItemModel>,
+    onCallButtonClicked: () -> Unit,
+    onEmailButtonClicked: () -> Unit,
+) {
+    val schedule = list[0].schedule
+    val address = list[0].address
+    val email = list[0].email
+    val phone = list[0].phone
+
+    Log.d("DATA_COMPOSE", "${schedule},${address},${email},${phone}")
+    //Get sizes from LeasePertTheme archive
+    val sizes = LeasePertTheme.sizes
+    Column(Modifier.fillMaxSize()) {
+        Text(
+            text = stringResource(R.string.LPInfo),
+            modifier = Modifier
+                .width(sizes.extraSize124)
+                .height(sizes.regularSize),
+            style = TextStyle(
+                textAlign = TextAlign.Justify,
+                color = MaterialTheme.colorScheme.tertiary,
+                fontFamily = FontFamily.SansSerif,
+                fontWeight = FontWeight.Bold,
+                fontSize = 20.sp,
+                lineHeight = 20.sp,
+                textIndent = TextIndent(firstLine = 14.sp)
+            )
+        )
+        Text(
+            text = address, style = TextStyle(
+                textAlign = TextAlign.Justify,
+                lineHeight = 20.sp,
+                textIndent = TextIndent(firstLine = 14.sp, restLine = 3.sp)
+            ), modifier = Modifier.padding(top = sizes.extraSize10)
+        )
+
+        Text(
+            text = stringResource(id = R.string.LPHours) + schedule,
+            modifier = Modifier.padding(top = sizes.extraSize10),
+            style = TextStyle(
+                textAlign = TextAlign.Justify,
+                lineHeight = 20.sp,
```

Comment:
Use the size file

Reply:
Where is the file for SP sizes? 
![imagen](https://github.com/irv205/droid_leaspert/assets/128720218/f7f4feae-6e18-4b4f-9dfa-b395c0c650f3)